portfolio/src/main/webapp/index.html

```diff
@@ -13,8 +13,8 @@
   <body onload = "parseSomething()">
     <nav>
 		<ul class="sticky">
```

Comment:
The indentation here (Line#15) also needs to be updated.
The difference in cloud shell and here might be caused by using tabs.

Reply:
The indentation is updated.